Speckle_Adapter/Speckle_Adapter.csproj

```
@@ -85,17 +93,17 @@
       <HintPath>..\..\BHoM_Engine\Build\Reflection_Engine.dll</HintPath>
       <Private>False</Private>
     </Reference>
-    <Reference Include="Rhinoceros_Engine">
-      <HintPath>..\..\Rhinoceros_Toolkit\Build\Rhinoceros_Engine.dll</HintPath>
+    <Reference Include="Reflection_oM">
+      <HintPath>..\..\BHoM\Build\Reflection_oM.dll</HintPath>
       <Private>False</Private>
     </Reference>
-    <Reference Include="RhinoCommon, Version=5.1.30000.16, Culture=neutral, PublicKeyToken=552281e97c755530, processorArchitecture=MSIL">
-      <HintPath>..\packages\RhinoCommon.5.12.50810.13095\lib\net35\RhinoCommon.dll</HintPath>
+    <Reference Include="Serialiser_Engine">
+      <HintPath>..\..\BHoM_Engine\Build\Serialiser_Engine.dll</HintPath>
       <Private>False</Private>
     </Reference>
     <Reference Include="SpeckleCore, Version=1.7.3.0, Culture=neutral, processorArchitecture=MSIL">
       <HintPath>..\packages\SpeckleCore.1.7.3.417\lib\netstandard2.0\SpeckleCore.dll</HintPath>
-      <Private>False</Private>
+      <Private>True</Private>
```

```suggestion
      <Private>False</Private>
```
I think we agreed this should be `false`. Along with SpeckleCoreGeometryClasses.
Sorry @alelom - can you double check. Thanks

---

Sorry - I got confused on the commit. Updated now.